src/FESpaces/DivConformingFESpaces.jl

```diff
@@ -0,0 +1,103 @@
+# This source file is though to put that code required in order to
+# customize ConformingFESpaces.jl to H(div)-conforming global FE Spaces built
+# out of RaviartThomas FEs. In particular, this customization is in the
+# definition of the shape functions (get_shapefuns) and the DoFs
+# (get_dof_basis) of the **global** FE space, which requires a sign flip
+# for those sitting on facets of the slave cell of the facet.
+
+function get_dof_basis(model::DiscreteModel,
+                       cell_reffe::AbstractArray{<:GenericRefFE{RaviartThomas}})
+    sign_flip = get_sign_flip(model, cell_reffe)
+    lazy_map(get_dof_basis,
+             cell_reffe,
+             get_cell_map(Triangulation(model)),
+             sign_flip)
+end
+
+function get_shapefuns(model::DiscreteModel,
+                       cell_reffe::AbstractArray{<:GenericRefFE{RaviartThomas}})
+    sign_flip = get_sign_flip(model, cell_reffe)
+    lazy_map(get_shapefuns,
+             cell_reffe,
+             get_cell_map(Triangulation(model)),
+             lazy_map(Broadcasting(ConstantField), sign_flip))
+end
+
+function get_sign_flip(model::DiscreteModel,
+                       cell_reffe::AbstractArray{<:GenericRefFE{RaviartThomas}})
+    lazy_map(get_sign_flip,
```

Comment:
(Minor comment). Perhaps, it is easier to pre compute the the cell-wise sign flip into a standard julia vector of `Int8` instead to try to make it lazy. It would be also significantly faster if you need to perform several loops over the vector, e.g. in non-linear problems

Reply:
The array is not as simple, it is an array of arrays of `Int`. So, I would not proceed this way.

Reply:
Then better lazy!

I would slightly refactor the definition of the kernel from which the lazy array is defined to match other similar kernels in Gridap. Here, the cell operation depends on a global state (the model). In this case, we usualy include the global state in a struct:

```julia
struct SignFlipMap{T} <: Map
  model::T
end

function return_cache(k::SignFlipMap,reffe,cell)
  model = k.model
 # same code as below
end

function evaluate!(cache,k::SignFlipMap,reffe,cell)
 # same code as below
end

# usage
lazy_map(SignFlipMap(model),cell_reffe,cell_ids)

```

Reply:
Done in 1685e4d99808277dba2e2f7062deb14e6f969bdb